### PR TITLE
Refactor/rename apiai to dialogflow

### PR DESCRIPTION
**Pull request by rokusottervanger**

### CODE REVIEW

File: src/dialogflow_ros/dialogflow_node.py [hunk 1]
```diff
@@ -11,19 +7,27 @@
...
         try:
-            CLIENT_ACCESS_TOKEN = rospy.get_param("~client_access_token")
-        except Exception as e:
+            self._client_access_token = rospy.get_param("~client_access_token")
+        except rospy.ROSException:
             rospy.logfatal("Missing required ROS parameter client_access_token")
             exit(1)
 
-        self.ai = apiai.ApiAI(CLIENT_ACCESS_TOKEN)
+        self.ai = apiai.ApiAI(self._client_access_token)
```
Comment (reinzor) [inline src/dialogflow_ros/dialogflow_node.py:27]: api dit not change?

**Review by reinzor — APPROVED** [review]
Seems good to me, btw, why are we refactoring this if we are not using this? What's the plan?

File: src/dialogflow_ros/dialogflow_node.py (hunk 1 above)
Comment (reinzor) [reply src/dialogflow_ros/dialogflow_node.py:27]: why is it still apiai?

Comment (rokusottervanger) [reply src/dialogflow_ros/dialogflow_node.py:27]: I guess they didn't want to break the API, so this is remains unchanged for now.

**Pull request merged**